GAE/src/com/gallatinsystems/gis/map/MapUtils.java

```diff
@@ -174,6 +175,11 @@ public static void recomputeCluster(Cache cache, SurveyedLocale locale, int delt
             }
         }
 
+        // delete locale if the Delta was a subtraction
+        SurveyedLocaleDao slDao = new SurveyedLocaleDao();
+        if (delta < 0) {
```

Comment:
Yep. The task with the -1 delta is created here:
https://github.com/akvo/akvo-flow/pull/1086/files#diff-a521baebc25d5bbfb464614e3eb1f5daR684

See comment.